lib/salus/scanners/yarn_audit.rb

```diff
@@ -28,7 +30,15 @@ def run
 
     def handle_latest_yarn_audit
       vulns = []
-      shell_return = run_shell(LATEST_YARN_AUDIT_COMMAND)
+
+      dep_types = @config.fetch('exclude_groups', [])
+      audit_command = if dep_types.include?('devDependencies')
```

Comment:
Can you update the docs at https://github.com/coinbase/salus/blob/master/docs/scanners/yarn_audit.md to convey the nuance w/what yarn 3 supports?  i.e. optional dependencies etc won't apply for yarn 3